Yashodha's org/Fare calculator API/service.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina/http;
+import ballerinax/java.jdbc;
+import ballerina/io;
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+
+type FlightDetails record {
+    string flightNumber;
+    string airline;
+    string flightDate;
+    string origin;
+    string destination;
+    int available;
+    int totalCapacity;
+    int baseFare;
+};
+
+service /fareCal on new http:Listener(9090) {
+
+    # A resource for generating greetings
+    # + name - the input string name
```

Comment:
Check the document comments, some the documented parameters are not present in the resource and not all the parameters in the resource is documented